app/assets/stylesheets/overrides.scss

```diff
@@ -225,12 +225,23 @@ ul.off-canvas-list li a:hover {
 .status-boxes {
   display: inline-block;
   width: 3%;
-  height: 10px;
+  height: 22px;
   margin: 0;
   padding-right: 0;
   padding: 0;
   margin-right: -5px;
 }
+
+.status-index {
+  color: navy;
+  overflow: visible;
```

Comment:
Properties should be ordered color, line-height, overflow, padding-left, position, top, z-index